Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -7377,6 +7377,19 @@ bool CStaticFunctionDefinitions::RemoveColPolygonPoint(CClientColPolygon* pColPo
     return false;
 }
 
+
+bool CStaticFunctionDefinitions::SetColPolygonHeight(CClientColPolygon* pColPolygon, float fFloor, float fCeil)
+{
+    if (pColPolygon->SetHeight(fFloor, fCeil))
+    {
+        RefreshColShapeColliders(pColPolygon);
+        return true;
+    }
+
+    return false;
+}
+
+
```

Comment:
Don't add to CStaticFunctionDefinitions when using the new parser, just add code directly to lua definitions